router.js

```diff
@@ -1,93 +1,501 @@
-////////////////////////////////////////////////////////////////////////////
-// Don't modify anything below here unless you know what you are doing :) //
-////////////////////////////////////////////////////////////////////////////
-var http = require('http'),
-	httpProxy = require('http-proxy'),
-	fs = require('fs'),
-	proxy = httpProxy.createProxyServer({}),
-	routerTable = {},
-	configFilePath,
-	serverPort;
-
-configFilePath = __dirname + '/config.json';
-
-var loadConfigData = function (callback) {
-	fs.readFile(configFilePath, 'utf8', function (err, data) {
-		if (err) { console.log('Error reading router config file: ' + err); } else {
-			data = JSON.parse(data);
-
-			routerTable = data.routerTable;
-
-			console.log('Router table config data updated successfully.');
+"use strict";
+
+const fs = require('fs')
+const http = require('http')
+const https = require('https')
+const urlParser = require('url')
+const httpProxy = require('http-proxy')
+const colors = require('colors')
+const sslConfig = require('ssl-config')('modern')
+
+/*
+const async = require('async')
+const crypto = require('crypto')
+const tls = require('tls')
+const //express = require('express')
+const spawn = require('child_process').spawn
+const spawnSync = require('spawn-sync')
+const path = require('path')
+const fsAccess = require('fs-access')
+const mkdirp = require('mkdirp')
+const configFilePath = __dirname + '/config.json'*/
+
```

Comment:
If this stuff is no longer needed, remove the code.

Reply:
I removed a lot and probably missed that part my bad